ui/src/components/harvest-games/HarvestGamesPage.js

```diff
@@ -0,0 +1,171 @@
+                lg:`${process.env.PUBLIC_URL}/images/harvest-games/planes_desktop.png`}}
+      height={{base:'fit-content',sm:'fit-content',md:'fit-content',lg:'fit-content'}}    
+      >
+      <Box  bgImage={{base:`${process.env.PUBLIC_URL}/images/harvest-games/cloud.gif`,
+                sm:`${process.env.PUBLIC_URL}/images/harvest-games/cloud.gif`,
+                md:`${process.env.PUBLIC_URL}/images/harvest-games/cloud.gif`,
+                lg:`${process.env.PUBLIC_URL}/images/harvest-games/cloud.gif`}}
+        bgPosition = 'bottom,left'
+        bgSize = {{base:'100%',sm:'100%',md:'45%',lg:'45%'}}
+        backgroundRepeat={{base:'no-repeat',sm:'no-repeat',md:'repeat-x',lg:'repeat-x'}}
+        >
+      {/*FLY HMCC Text*/}
+      <Box
+        objectPosition={'center'}
+        height = '10'
+        bgColor='none'
+        paddingTop = {{base: '2%',sm:'1%', md:'1%', lg:'2%',xl:'4%'}}
+        paddingLeft = {{base: '5%',sm:'5%', md:'24%', lg:'30%', xl:'34%'}}
+        paddingRight= {{base: '5%', sm:'10%', md:'35%', lg:'35%', xl: '30%'}}>
+        <Image
+          src= {`${process.env.PUBLIC_URL}/images/harvest-games/FLYHMCC_DESKTOP.png`}
+          bgColor = 'none'
+          paddingLeft = '20px' 
+          objectFit='cover'
+          /> 
+      </Box>
+
+      <Container maxW="80%" 
+      paddingTop = {{base: '35%',sm:'35%', md:'20%', lg:'17%', xl: '15%'}}
+      paddingBottom = '5%'
+      height = {{base: '70%',sm:'90%', md:'90%', lg:'100%', xl: '100%'}}
+      bgColor = "none"
+      >
+      <Box
+          background="none"
+          paddingTop = {{base: '5%',sm:'1%', md:'1%', lg:'2%'}}
+          
+          height = '90%'> {/*important so that panels do not overflow*/}
+         
+        {/* DESKTOP */}
+
+          <Tabs 
+          orientation='vertical'
+          display={['none','none','flex']}
+          borderBottom="CaptionText"
+          borderRadius="5px"
+          variant="unstyled"
+          mb="5%"
+          default={1}
+          >
+
+          <TabList border="none" width = '250px' paddingRight={"10"}>
+            <Tab 
+              borderBottom="5px solid #E2E8F0" 
+              _selected={{ borderColor: '#0628A3', textColor: "#3A6693", fontWeight: 'bold' }}>
+              <Text>
+                Leaderboard
+              </Text>
+              </Tab>
+            <Tab 
+              borderBottom="5px solid #E2E8F0"
+              _selected={{ borderColor: '#0628A3', textColor: "#3A6693", fontWeight: 'bold'}}>Pre-Games</Tab>
+          </TabList>     
+
+          <TabPanels bgColor='#FFFFFF' borderRadius={'10'} bottomPadding = "50px" >
+            <TabPanel  height = "40em">
+            <Text
+              fontSize = {{base: '20px',sm:'25px', md:'30px', lg:'40px'}}
+              textAlign = 'center'
+            >
+              Leadership BOARD: 
+              </Text>
+            </TabPanel>
+            
+            <TabPanel  height = "80em">
```

Comment:
i think we don't need to make it this tall, maybe just make it `minHeight="80vh"` to make it kinda fill screen since there's no actual content